code/pizzabot.js

```diff
@@ -4,4 +4,88 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+var x = vegetarian
```

Comment:
Try to use `const` or `let` before using `var` when declaring a variable. In this case `const` would suit perfect! Another thing is the naming could be better. A name should be a description of what it is. Like in this example `x` doesn't say that it is a vegetarian pizza. 
```suggestion
const vegetarian = "vegetarian pizza"
```